index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html> 
+<html>
+<head> 
+    <title>Hugo Padilla's Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="Hugo Padilla">
+</head>
+
+<body>
+    <h1>Hugo Padilla</h1>
+
+    <section id = "About">
```

Comment:
avoid spaces between attributes and their value `<section id="About">`

Talking about that make sure to use third-party VS code extension I believe they can help you with that, extension like prettier can be really useful